stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Jennifer Ding:</p><ul><li>What an interesting conclusion - it's fascinating to think about how unique physical artefacts may not be a good fit for analysis via data-driven ML methods that require many prior, similar examples. To me it seems to raise the question of what kind of data and historical artefact is fit for ML problems.</li></ul><p>Joe Cerniglia:</p><p>That’s encouraging. Thank you. The other conclusion I am hoping that people will draw is that machine learning models are not ‘sentient’ or ‘omniscient’, an idea that seems to be taking hold in the popular imagination. I would contend that sometimes what they can’t learn can tell us as much about the objects of their analysis as what they can learn.&nbsp;People have often confused this jar with something they could purchase in their local CVS or Boots. I believe the elemental lab results, and the incorrect ML predictions, help dispel that notion. This jar is something very different from the typical flotsam I noticed on Nikumaroro, or anything I have ever seen in a store, or eBay, or anywhere else, and I’ve looked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>